### ISSUE TITLE
No installed version detected with npm lockfileVersion 3

### ISSUE THREAD

**Issue by lpgera** [browserslist/update-db#18]
Hi,

I noticed that when running the update in a project which has an npm `package-lock.json` with [version 3](https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json#lockfileversion), the output looks like this:

```
Latest version:     1.0.30001478
Installed version:  none
[...]
```

So even though I have `caniuse-lite` installed, the installed version is not detected. Note, that the update is still successful, so this detection error doesn't prevent the script from updating the db.

---

Version 3 is the default lock file format for `npm@9.x`, and `npm@9.x` comes pre-bundled with `node@18` since February, I guess it's time to get ready for parsing the new format.

I think the biggest difference between the formats is this:
* Version 1 has a `dependencies` property on the top level of the JSON.
* Version 2 has both a `dependencies` and a `packages` property on the top level. The two seems to be redundant, I guess for backwards compatibility.
* Version 3 breaks the backward compatibility and it only has a `packages` property on the top level.

The content of `dependencies` and `packages` probably also differ, but I haven't dug deep enough yet to understand all the nuances.

**Comment by ai** [comment]
Should be fixed in 1.0.11.

Can you try?

**Comment by lpgera** [comment]
That was very quick! 👏 Works like a charm now, thank you.
